elixir/.gitignore

```diff
@@ -0,0 +1,23 @@
+# The directory Mix will write compiled artifacts to.
+/_build
+
+# If you run "mix test --cover", coverage assets end up here.
+/cover
```

Comment:
I've never seen `/.fetch` myself, but it's in the [default template](https://github.com/elixir-lang/elixir/blob/master/lib/mix/lib/mix/tasks/new.ex#L250) generated by `mix new`.

I suggest erring on **not** ignoring `*.ez`, since it's possible for Mix archives to be included in deploys.